packages/ethereum-storage/src/ignored-dataIds.ts

```diff
@@ -0,0 +1,152 @@
+import * as Keyv from 'keyv';
+
+import { StorageTypes } from '@requestnetwork/types';
+
+/**
+ * Allows to save and retrieve the dataIds ignored with the reason
+ */
+export default class IgnoredDataIds {
+  /**
+   * Store the reason we ignored data ids in a dictionary
+   */
+  public ignoredDataIds: Keyv<StorageTypes.IIgnoredDataId>;
+
+  public listIgnoredDataIds: Keyv<string[]>;
+
+  /**
+   * Constructor
+   * @param store a Keyv store to persist the metadata
+   */
+  public constructor(store?: Keyv.Store<any>) {
+    this.ignoredDataIds = new Keyv<StorageTypes.IIgnoredDataId>({
+      namespace: 'dataIdIgnored',
+      store,
+    });
+
+    this.listIgnoredDataIds = new Keyv<string[]>({
```

Comment:
I don't understand why we need this and what the code related to this do :(

Reply:
I ve added a comment and JIRA it.